### PR TITLE
[PostgreSql] Added connection test cmdlet 

### PR DESCRIPTION
**Pull request by DaeunYim**
## Description

- Added Test-AzPostgreSqlFlexibleServerConnect cmdlet to test connection and run a simple query
- Implemented a workaround to resolve the resource id bug

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/PostgreSql/Az.PostgreSql.psd1 [hunk 1]
```diff
@@ -11,7 +11,7 @@
   DotNetFrameworkVersion = '4.7.2'
   RequiredAssemblies = './bin/Az.PostgreSql.private.dll'
   FormatsToProcess = './Az.PostgreSql.format.ps1xml'
-  FunctionsToExport = 'Get-AzPostgreSqlConfiguration', 'Get-AzPostgreSqlConnectionString', 'Get-AzPostgreSqlFirewallRule', 'Get-AzPostgreSqlReplica', 'Get-AzPostgreSqlServer', 'Get-AzPostgreSqlVirtualNetworkRule', 'New-AzPostgreSqlFirewallRule', 'New-AzPostgreSqlReplica', 'New-AzPostgreSqlServer', 'New-AzPostgreSqlVirtualNetworkRule', 'Remove-AzPostgreSqlFirewallRule', 'Remove-AzPostgreSqlServer', 'Remove-AzPostgreSqlVirtualNetworkRule', 'Restart-AzPostgreSqlServer', 'Restore-AzPostgreSqlServer', 'Update-AzPostgreSqlConfiguration', 'Update-AzPostgreSqlFirewallRule', 'Update-AzPostgreSqlServer', 'Update-AzPostgreSqlVirtualNetworkRule', '*'
```
Comment (msJinLei) [inline src/PostgreSql/Az.PostgreSql.psd1:14]: In mysql, you don't update psd1. You can add the updated Az.MySql.psd1 here

File: src/PostgreSql/utils/adjectives.txt [hunk 2]
```diff
@@ -1,166 +0,0 @@
-aboard
```
Comment (msJinLei) [inline src/PostgreSql/utils/adjectives.txt:1]: Do you want to delete these 2 files?

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/PostgreSql/Az.PostgreSql.psd1 (hunk 1 above)
Comment (DaeunYim) [reply src/PostgreSql/Az.PostgreSql.psd1:14]: I thought the auto-generation will automatically upgrade this file. Let me add it here.

File: src/PostgreSql/utils/adjectives.txt (hunk 2 above)
Comment (DaeunYim) [reply src/PostgreSql/utils/adjectives.txt:1]: I moved them to the custom folder.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**